docs/kubernetes/kctlr-ingress.rst

```diff
@@ -5,195 +5,187 @@
 
-\
+IP address assignment
+`````````````````````
 
-.. table:: Tasks
+The |kctlr| supports the following options for IP address assignment. See the `k8s-bigip-ctlr configuration parameters`_ table for more information about these settings.
 
-   =======  ===================================================================
-   Step     Description
-   =======  ===================================================================
-   1.       :ref:`Create a BIG-IP Self IP address <ingress self IP>` for the
-            virtual server.
-   2.       :ref:`create k8s ingress`.
-   3.       :ref:`add health monitor to ingress`.
-   4.       :ref:`deploy ingress resource`.
-   5.       :ref:`verify-ingress-vs-created`.
-   =======  ===================================================================
+.. _ingress default IP:
 
+Default IP address
+~~~~~~~~~~~~~~~~~~
 
-.. _ingress self IP:
+.. include:: /_static/reuse/k8s-version-added-1_4.rst
 
-Initial Setup
--------------
+You can set a default IP address for Ingress resources. To do so:
 
-#. Create a BIG-IP Self IP address.
+#. Add the ``--default-ingress-ip`` config parameter to the :ref:`k8s-bigip-ctlr Deployment <k8s-bigip-ctlr-deployment>`.
+#. Add the annotation ``virtual-server.f5.com/ip="controller-default"`` to your Ingress resource.
 
-   Allocate a `Self IP address`_ from the external network on the BIG-IP system. This is the IP address you should assign to the Ingress' virtual server.
+The |kctlr| will replace "controller-default" with the IP address provided for the ``default-ingress-ip`` parameter.
 
-.. todo:: add instructions for creating Self IP on BIG-IP; reuse? or link to in VE public cloud docs?
+When you define a default ingress IP address, each Ingress resource configured to use the "controller-default" IP shares the same BIG-IP virtual server. The |kctlr| attaches a separate policy to the virtual server for each Ingress to ensure correct traffic routing for those resources.
 
+.. important::
 
-.. _ingress-quick-start:
+   You can only define one ``--default-ingress-ip`` per |kctlr| instance.
 
-Quick Start
------------
+If you're using multiple Controllers to monitor separate namespaces, you can define a default IP address for each Controller. This type of deployment allows you to isolate the VIPs in each namespace from each other.
 
-You can add the `supported Ingress annotations`_ to any existing Ingress resource using :command:`kubectl annotate` or :command:`oc annotate`.
-At minimum, define the following properties:
+DNS lookup
+~~~~~~~~~~
 
-- :code:`virtual-server.f5.com/ip`
-- :code:`virtual-server.f5.com/partition`
+The |kctlr| uses DNS lookup to resolve hostnames by default (as of v1.3.0). Provide the hostname in the ``virtual-server.f5.com/ip=`` annotation. The Controller will attempt to resolve the hostname and assign the resolved host's IP address to the Ingress' virtual server.
```

Comment:
Same comment as above; the hostname comes from the first host in the Ingress spec, not the annotation.